TAF/testScenarios/functionalTest/V2-API/core-data/event/PUT-positive.robot

```diff
@@ -0,0 +1,16 @@
+*** Settings ***
+Resource         TAF/testCaseModules/keywords/commonKeywords.robot
+Suite Setup      Setup Suite
+
+*** Variables ***
+${SUITE}         Core-Data Event PUT Postive Testcases
+
+*** Test Cases ***
+EventPUT001 - Update event pushed time
+    Given Create Multiple Events
+    When Update Multiple Events
+    Then Should Return Status Code "207"
+    And Should Have Content-Type "application/json"
+    And Each Item Should Contain Status Code "200" and checksum
+    And Each Item Pushed Time Should Be Updated
+    [Teardown]  Delete Events
```

Comment:
Suggest merging the tests for PUT to the same file because the API return code always returns "207".